### PR TITLE
Update spanish.py

### PR DESCRIPTION
**Pull request by MarcoAurelioWM**
added a couple of words, defined few and removed one which is not a bad word "per se".

### CODE REVIEW

File: revscoring/languages/spanish.py [hunk 1]
```diff
@@ -135,15 +134,16 @@
...
     r"me[- ]?castra[- ]?la[- ]?madre",  # 'I <something> your mother' ???
-    r"me[- ]?la[- ]?pelan?s?",  # "You ain't shit" ???
+    r"me[- ]?la[- ]?pelan?s?",  # I don't give a f***
     r"mi?er[dg]a+s?\w*",  # shit
     r"mimsn",  # ???
     r"ming[ao]",  # ugly person
     r"mocos",  # boogers / mucus
     r"mojon",  # shit
     r"monda",  # man part / penis
+    r"mongolo", # idiot (slang)
```
Comment (halfak) [inline revscoring/languages/spanish.py]: flake8 is failing here.  There should be two spaces before the comment "#"

File: revscoring/languages/spanish.py [hunk 2]
```diff
@@ -183,6 +183,7 @@
     r"sida",  # AIDS
     r"sorete",  # piece of shit
     r"sudaca",  # greaser
+    r"subnormal", # retarded
```
Comment (halfak) [inline revscoring/languages/spanish.py]: Needs another space here too.  Then all should be good :)

File: revscoring/languages/spanish.py (hunk 1 above)
Comment (MarcoAurelioWM) [reply revscoring/languages/spanish.py]: Fixing.

File: revscoring/languages/spanish.py (hunk 2 above)
Comment (MarcoAurelioWM) [reply revscoring/languages/spanish.py]: Fixing those. Thanks.

**Comment by halfak** [comment]
Thanks!

**Pull request merged**